Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/impl/AuditServiceImpl.java

```diff
@@ -42,37 +70,11 @@
 public class AuditServiceImpl implements AuditService {
 
     private static final Logger LOG = LoggerFactory.getLogger(AuditServiceImpl.class);
+    private static final String EVENT_DATE_FORMAT = "MM/dd/yyyy HH:mm:ss";
+    private AuditRetrieve auditRetrieve = null;
 
-    @Override
-    public ArrayList<Audit> createMockAuditRecord() {
-        Audit audit1 = new Audit();
-        audit1.setEventOutcomeIndicator("Success");
-        audit1.setEventType("QueryForDocuments");
-        audit1.setMessageId("MessageId-1");
-        audit1.setRemoteHcid("2.2");
-        audit1.setUserId("Thomas");
-
-        Audit audit2 = new Audit();
-        audit2.setEventOutcomeIndicator("Success");
-        audit2.setEventType("RetrieveDocuments");
-        audit2.setMessageId("MessageId-2");
-        audit2.setRemoteHcid("3.3");
-        audit2.setUserId("Joe");
-        try {
-            audit1.setEventTimestamp(getTimeStamp());
-            audit2.setEventTimestamp(getTimeStamp());
-        } catch (ParseException ex) {
-            LOG.error("Timestamp  formatting Exception: " + ex.getLocalizedMessage(), ex);
-        }
-        ArrayList<Audit> auditRecord = new ArrayList<>();
-        auditRecord.add(audit1);
-        auditRecord.add(audit2);
-        return auditRecord;
-    }
-
-    private Timestamp getTimeStamp() throws ParseException {
-        Date date = new Date();
-        return new Timestamp(date.getTime());
+    public AuditServiceImpl() {
+        auditRetrieve = new AuditQueryLogProxyObjectFactory().getAuditRetrieveProxy();
     }
 
     public String createMockAuditMessage(long id) {
```

Comment:
Assuming this mock method will be gone with ticket addressing auditId search fix